embulk-standards/src/main/java/org/embulk/standards/CsvParserPlugin.java

```diff
@@ -261,7 +260,7 @@ public void run(TaskSource taskSource, final Schema schema,
                             public void booleanColumn(Column column)
                             {
                                 String v = nextColumn();
-                                if (v == null) {
+                                if (v == null || v.isEmpty()) {
```

Comment:
I agree 1 is technically the "correct" behavior.
I re-thought for use cases of 2, and I could find no use case for it.
I changed the behavior to 1.